docs/design/agent-config-editing/research/rfc.html

```
@@ -0,0 +1,1063 @@
+<title>RFC: The agent edits its own configuration</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)